frame/transaction-payment/src/lib.rs

```diff
@@ -227,12 +237,18 @@ where
 }
 
 /// A struct to make the fee multiplier a constant
-pub struct ConstFeeMultiplier<M: Get<Multiplier>>(sp_std::marker::PhantomData<M>);
+pub struct ConstFeeMultiplier<M: Get<Multiplier>, X: Get<Multiplier>>(
+	sp_std::marker::PhantomData<M>,
+	sp_std::marker::PhantomData<X>,
+);
 
-impl<M: Get<Multiplier>> MultiplierUpdate for ConstFeeMultiplier<M> {
+impl<M: Get<Multiplier>, X: Get<Multiplier>> MultiplierUpdate for ConstFeeMultiplier<M, X> {
```

Comment:
This does not need a separate `X`. The min and max for `ConstFeeMultiplier` are both the same and just `M`.